prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -22,184 +23,189 @@ elements:
     csvFields: [{name: Testing_lab_zip_code}]
 
-  - name: standard.testing_lab_phone_number
+  - name: testing_lab_phone_number
     csvFields: [{name: Testing_lab_phone_number}]
 
-  - name: standard.testing_lab_county
+  - name: testing_lab_county
     csvFields: [{name: Testing_lab_county}]
 
   - name: organization_name
     documentation: For cases where organization owns many facilities (eg, a large hospital system)
     csvFields: [{name: Organization_name}]
     type: TEXT
 
-  - name: standard.ordering_facility_name
+  - name: ordering_facility_name
     csvFields: [{name: Ordering_facility_name}]
 
-  - name: standard.ordering_facility_street
+  - name: ordering_facility_street
     csvFields: [{name: Ordering_facility_street}]
 
-  - name: standard.ordering_facility_street2
+  - name: ordering_facility_street2
     csvFields: [{name: Ordering_facility_street_2}]
 
-  - name: standard.ordering_facility_city
+  - name: ordering_facility_city
     csvFields: [{name: Ordering_facility_city}]
 
-  - name: standard.ordering_facility_state
+  - name: ordering_facility_state
     csvFields: [{name: Ordering_facility_state}]
 
-  - name: standard.ordering_facility_zip_code
+  - name: ordering_facility_zip_code
     csvFields: [{name: Ordering_facility_zip_code}]
 
-  - name: standard.ordering_facility_phone_number
+  - name: ordering_facility_phone_number
     csvFields: [{name: Ordering_facility_phone_number}]
 
-  - name: standard.ordering_facility_county
+  - name: ordering_facility_county
     csvFields: [{name: Ordering_facility_county}]
 
-  - name: standard.ordering_provider_id
+  - name: ordering_provider_id
     csvFields: [{name: Ordering_provider_ID}]
 
-  - name: standard.ordering_provider_last_name
+  - name: ordering_provider_last_name
     csvFields: [{name: Ordering_provider_last_name}]
 
-  - name: standard.ordering_provider_first_name
+  - name: ordering_provider_first_name
     csvFields: [{name: Ordering_provider_first_name}]
 
-  - name: standard.ordering_provider_street
+  - name: ordering_provider_street
     csvFields: [{name: Ordering_provider_street}]
 
-  - name: standard.ordering_provider_street2
+  - name: ordering_provider_street2
     csvFields: [{name: Ordering_provider_street_2}]
 
-  - name: standard.ordering_provider_city
+  - name: ordering_provider_city
     csvFields: [{name: Ordering_provider_city}]
 
-  - name: standard.ordering_provider_state
+  - name: ordering_provider_state
     csvFields: [{name: Ordering_provider_state}]
 
-  - name: standard.ordering_provider_zip_code
+  - name: ordering_provider_zip_code
     csvFields: [{name: Ordering_provider_zip_code}]
 
-  - name: standard.ordering_provider_phone_number
+  - name: ordering_provider_phone_number
     csvFields: [{name: Ordering_provider_phone_number}]
 
-  - name: standard.ordering_provider_county
+  - name: ordering_provider_county
     csvFields: [{name: Ordering_provider_county}]
 
+  - name: reporting_facility_name
```

Comment:
?   This brings up a question I've had for a while:   mappers tautologically work in one direction only, what does it mean when a schema is used for both incoming and outgoing data.    Do we need a 'reverse' mapper in some cases?

In the case here, SimpleReport is not currently sending us a `reporting_facility_name`, but the mapper tells how to create it from other data -- all good.   But what about when reading back in data _generated_ using this schema.   Now the `reporting_facility_name` field exists - shouldn't that trigger a "no-op" or "passthru-as-is" mapper?


You can see this more clearly in the example of how the AZ schema uses, but does not contain, the organization_name, which is only in the pdi schema.    AZ's schema says: 
```
   # Nonstandard
   - name: organization_and_facility
     mapper: 'concat(organization_name, ordering_facility_name)'
     csvFields: [{name: Ordering_facility_name}]
```

But AZ data, as created by the hub _no longer contains the organization_name column!_  So, if that data is read back into the Hub, this mapper, if it runs, should fail.   That is not happening, so I think there's an implied "no-op" or "passthru-as-is" mapper in that situation as well.